docs/_includes/date-textbox.html

```diff
@@ -42,10 +41,111 @@ <h3>September 2019</h3>
             </div>
             <div class="calendar__body">
               <div class="calendar__month">
-                {% include calendar-month.html index="1" interactive="true" %}
+                <table>
+                  <caption>August 2023</caption>
+                  <thead>
+                      <tr>
+                          <th>Su</th><th>Mo</th><th>Tu</th><th>We</th><th>Th</th><th>Fr</th><th>Sa</th>
+                      </tr>
+                  </thead>
+                  <tbody>
+                      <tr>
+                          <td colspan="2"></td>
+                          <td><span>1</span></td>
```

Comment:
This should be an interactive calendar, we don't support static calendars in date textbox